### PR TITLE
Add blog post for Cake 2.0

### PR DESCRIPTION
**Pull request by devlead**
* v2.0.0  Blog Post

### CODE REVIEW

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+﻿---
+title: Cake v2.0.0 released
+category: Release Notes
+author: devlead
+---
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:5]: Add `releaseName` meta data to update version on home page and link to this blog post

```suggestion
releaseName: 2.0.0
---
```

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+Version 2.0.0 of Cake has been released.
+
+This version is the 119th release of Cake, but it's just not any release - say hello to Cake 2.0 🎉
+In this blog post, we will go through some of the major new features of 2.0, which platforms are supported, what to consider when upgrading to Cake 2.0.
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:10]:
```suggestion
In this blog post, we will go through some of the major new features of 2.0, which platforms are supported and what to consider when upgrading to Cake 2.0.
```

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+But first of all, we would like to thank our amazing community, we wouldn't be where we are today without you, so huge thanks to you all!❤
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:12]:
```suggestion
But first of all, we would like to thank our amazing community, we wouldn't be where we are today without you, so huge thanks to all of you! ❤
```

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+We've now set the recommended version of [Cake.Core](https://www.nuget.org/packages/Cake.Core) for addins to target to 2.0.
+
+This won't break the build but you might see warnings like below when addins or modules are loaded.
+
+```bash
+The assembly 'Cake.FileHelpers, Version=4.0.1.0, Culture=neutral, PublicKeyToken=null'
+is referencing an older version of Cake.Core (1.0.0).
+```
+
+### Updated docker images
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:102]:
```suggestion
### Updated Docker images
```

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+
+Images are now continuously built for the last 10 stable versions of Cake version 1.0 or newer. Currently based on official Microsoft images available ( [mcr.microsoft.com/dotnet/sdk](https://mcr.microsoft.com/dotnet/sdk) ).
+
+To pin to a specific version suffix with Cake version i.e. cakebuild/cake:sdk-6.0 becomes cakebuild/cake:sdk-6.0-v2.0.0.
+
+Tags are added dynamically as new are added to Microsoft container registry, check [hub.docker.com/r/cakebuild/cake/tags](https://hub.docker.com/r/cakebuild/cake/tags) for currently available tags.
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:108]:
```suggestion
Tags are added dynamically as new are added to Microsoft container registry.
Check [hub.docker.com/r/cakebuild/cake/tags](https://hub.docker.com/r/cakebuild/cake/tags) for currently available tags.
```

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+
+Pull-requests and issues for the docker images can be raised in Cake's docker repo: [github.com/cake-build/docker](https://github.com/cake-build/docker)
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:110]:
```suggestion
Pull requests and issues for the Docker images can be raised in [github.com/cake-build/docker](https://github.com/cake-build/docker).
```

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:111]: Misses a heading

```suggestion

### Contributions

```

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 8]
```diff
@@ -0,0 +1,226 @@
...
+- [nils-a](https://github.com/nils-a)
+- [norpache](https://github.com/norpache)
+- [pascalberger](https://github.com/pascalberger)
+- [sgronlund](https://github.com/sgronlund)
+- [SIkebe](https://github.com/SIkebe)
+- [ToMee99](https://github.com/ToMee99)
+
+Full details of everything that was included in this release can be seen below.
+
+<!--excerpt-->
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:135]: Remove (there's already one above)

File: input/blog/2021-11-30-cake-v2.0.0-released.md (hunk 7 above)
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:111]: Should we add another chapter that Azure DevOps Build Task Extension has been deprecated? Or do we want to announce this in a separate blog post?

**Review by pascalberger — CHANGES_REQUESTED** [review]

File: input/blog/2021-11-30-cake-v2.0.0-released.md (hunk 7 above)
Comment (devlead) [reply input/blog/2021-11-30-cake-v2.0.0-released.md:115]: Think we can do a separate follow-up one, if you have something ready you can add it as a suggestion.

Comment (patriksvensson) [reply input/blog/2021-11-30-cake-v2.0.0-released.md:115]: Perhaps we should we show all contributors here for Cake 1.0 to 2.0?

Comment (devlead) [reply input/blog/2021-11-30-cake-v2.0.0-released.md:115]: > Perhaps we should show all contributors here for Cake 1.0 to 2.0?

@patriksvensson good idea, added.

File: input/blog/2021-11-30-cake-v2.0.0-released.md [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+
+As a consequence of no longer shipping [Cake runner for .NET Framework], we will also stop shipping the [Cake.Portable Chocolatey package]
+and [Homebrew Cake formulae].
+
+[Cake runner for .NET Core] has been deprecated since version 1.0 with [Cake .NET Tool] as the suggested replacement.
+For users of [Cake runner for .NET Framework], it is also suggested to switch to [Cake .NET Tool] and run builds on .NET Core 3.1 or newer.
+For users which rely on an extension, or other dependencies, which require .NET Framework or .NET Core 3.0 or older, the suggestion is to stay on Cake 1.x.
+
+See [Sunsetting of .NET Framework and .NET Core runners in Cake 2.0] for details.
+
```
Comment (pascalberger) [inline input/blog/2021-11-30-cake-v2.0.0-released.md:63]: Links are missing

```suggestion

[Cake runner for .NET Framework]: /docs/running-builds/runners/cake-runner-for-dotnet-framework
[Cake runner for .NET Core]: /docs/running-builds/runners/cake-runner-for-dotnet-core
[Cake .NET Tool]: /docs/running-builds/runners/dotnet-tool
[Cake Frosting]: /docs/running-builds/runners/cake-frosting
[Cake.Portable Chocolatey package]: https://community.chocolatey.org/packages/cake.portable
[Homebrew Cake formulae]: https://formulae.brew.sh/formula/cake
[Sunsetting of .NET Framework and .NET Core runners in Cake 2.0]: /blog/2021/10/sunsetting-runners
```

**Review by pascalberger — CHANGES_REQUESTED** [review]
Some links are missing. Otherwise LGTM

**Review by pascalberger — APPROVED** [review]

**Review by augustoproiete — APPROVED** [review]
LGTM

**Pull request merged**